api/models.go

```diff
@@ -188,9 +188,30 @@ type MobileContentUnitResponse struct {
 	Items []*MobileContentUnitResponseItem `json:"items"`
 }
 
+type MobileSearchResponse struct {
+	Total int                         `json:"total"`
```

Comment:
Why you use int and int32 and not int64?

Reply:
fixed